charts/fpl-case-service/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: fpl-case-service
 apiVersion: v1
```

Comment:
In the Spring boot 2.3.0 upgrade message it says:
You will need to move to apiVersion: v2  in Chart.yaml for the helm chart as well, see the below sample PRs.

Do we need to do this now too? @tomaszpowroznik maybe you can advise

Reply:
I think we can change to v2, but it is not really related to upgrading java chart.
Helm doc says:
The apiVersion field should be v2 for Helm charts that require at least Helm 3. Charts supporting previous Helm versions have an apiVersion set to v1 and are still installable by Helm 3.

Looks like not mandatory but better to change